cli/src/testing/run.ts

```diff
@@ -23,25 +25,28 @@ export const runTests = async (
     endpoint,
     host,
     verbose,
-  }: { endpoint: string; host: string; verbose: boolean },
+    env,
+  }: { endpoint: string; host: string; verbose: boolean; env: string },
 ) => {
+  let initEnv: { [key: string]: string } = {}
+  if (env) {
+    initEnv = dotenv.parse(fs.readFileSync(env, "utf8"))
```

Comment:
since this is completely user populated, can we have a check on the file content ? If dotenv.parse can't parse something, it doesn't raise an error, but silently lets it pass.

Reply:
status : not validating env, but rather listing the count of env items loaded from path